### PR TITLE
Selenium

### PR DESCRIPTION
**Pull request by Mexaby**

### CODE REVIEW

File: SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs [hunk 1]
```diff
@@ -0,0 +1,349 @@
...
+            driver.FindElement(By.CssSelector("#header > div > div.skip-links > div > a > span.label")).Click();
+            driver.FindElement(By.CssSelector("#header-account > div > ul > li.last > a")).Click();
+
+            //fill in the fields and click login
+            driver.FindElement(By.Id("email")).SendKeys("asd@asd.com");
+            driver.FindElement(By.Id("pass")).SendKeys("asdasd");
+            driver.FindElement(By.Id("send2")).Click();
+
+            //access a sub-category
+            Actions action = new Actions(driver);
```
Comment (ramonaZmole) [inline SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs:254]: same. use var when declaring local variables

File: SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs [hunk 2]
```diff
@@ -0,0 +1,349 @@
...
+        {
+            //initialize driver
+            WebDriver driver = new ChromeDriver();
+            driver.Manage().Window.Maximize();
+
+            //go to main page
+            driver.Navigate().GoToUrl("http://qa2magento.dev.evozon.com/");
+
+            //access women's main category and return to home page
+            driver.FindElement(By.CssSelector("#nav > ol > li.level0.nav-1.first.parent > a"));
```
[the remark this answers is not in the document]
Comment (Mexaby) [reply SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs:86]: Fixed

File: SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs [hunk 3]
```diff
@@ -0,0 +1,349 @@
...
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Interactions;
+
+namespace SeleniumWebDriver
+{
+    [TestClass]
+    public class Tests
+    {
+        [TestMethod]
+        public void ValidLogin()
```
[the remark this answers is not in the document]
Comment (Mexaby) [reply SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs:11]: Renamed methods to be more suggestive

File: SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+            driver.FindElement(By.CssSelector("#header > div > div.skip-links > div > a > span.label")).Click();
+            driver.FindElement(By.CssSelector("#header-account > div > ul > li.last > a")).Click();
+
+            //fill in the fields and click login
+            driver.FindElement(By.Id("email")).SendKeys("asdf@asdf.com");
+            driver.FindElement(By.Id("pass")).SendKeys("111111");
+            driver.FindElement(By.Id("send2")).Click();
+
+            //access a sub-category
+            Actions action = new Actions(driver);
```
[the remark this answers is not in the document]
Comment (Mexaby) [reply SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs:210]: Replaced local variable declarations with var

File: SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs [hunk 5]
```diff
@@ -0,0 +1,349 @@
...
+            IWebElement accessoriesHover = driver.FindElement(By.CssSelector("#nav > ol > li.level0.nav-3.parent > a"));
```
[the remark this answers is not in the document]
Comment (Mexaby) [reply SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs:211]: Renamed all occurrences

File: SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs (hunk 1 above)
Comment (Mexaby) [reply SeleniumWebDriver/SeleniumWebDriver/UnitTest1.cs:254]: Already done

**Comment by Mexaby** [comment]
Will create another pull request with the updated code.

**Pull request closed without merging**